rudder-web/src/main/scala/com/normation/rudder/web/components/RuleEditForm.scala

```diff
@@ -228,6 +226,7 @@ class RuleEditForm(
             | $$("#editRuleZone").bind( "show", function(event, ui) {
             | if(ui.panel.id== 'ruleComplianceTab') { ${ruleComplianceTabAjax}; }
             | });
+            |${Replace("details", new RuleCompliance(rule,directiveLib, nodeInfos, rootRuleCategory).display).toJsCmd};
```

Comment:
OK, that's nice. Moreover, if it takes too long, it won't block the "update form" part, and let user do stuff in the meantime.